src/api/routes/meetings.ts

```diff
@@ -113,9 +114,49 @@ const decline = procedure
     商量解决方案。`, baseUrl);
 });
 
+/**
+ * List all meeting slots for admin management.
+ * Only MentorshipManager can access this.
+ */
+const listSlots = procedure
+  .use(authUser("MentorshipManager"))
+  .query(async () => {
```

Comment:
follow the same format as other routes where the bracket start at column 0 of next line